SRTNet.h

```diff
@@ -242,6 +242,16 @@ class SRTNet {
      */
     SRTSOCKET getBoundSocket() const;
 
+    /**
+     *
+     * @brief Get the listening port of the server socket. Useful If the input port was 0 when configuring the server,
+     * this function will then return which port that is being used.
+     * @returns The listening port of the server in case it has been assigned, otherwise 0. If the instance is in
+     * client mode 0 will be returned as well.
+     *
+     */
+    uint16_t getListeningPortServer() const;
```

Comment:
Would it be an idea to make this even more general and return the locally bound port (i.e. same as this implementation for Server/Listener mode, but add the functionality to return the locally used port in Client/Caller mode)? That way, I think we can replace this piece of code in the OutputInterfaces file:
https://github.com/edgeware/stardom-core/blob/main/production_pipeline/sources/OutputInterfaces.cpp#L199-L214

Reply:
Hm I guess that is the original code I've copied, to later be replaced by this function.. but I thought I didn't get that to work for client-mode properly.. will re-test that and then we can rename the function (original idea was to make it work for client and server :) )

Reply:
Got it working now, great, will re-name the function to getBoundPort() then?

Reply:
Or even `getLocallyBoundPort`?